frappe/patches/v10_0/delete_desktop_icon_from_doctype.py

```diff
@@ -0,0 +1,6 @@
+import frappe
+
+def execute():
+	for doc in frappe.get_all("Doctype", fields=["name", "module"], filters = {"icon":("not like", "")}):
+		frappe.reload_doc(doc.module, "doctype", doc.name)
+		frappe.db.set_value("Doctype", doc.name, "icon", "")
```

Comment:
haha not so easy.. .they have to be removed from all json files too

Reply:
@rmehta hmm which json files, sorry not following you.

Reply:
https://github.com/frappe/erpnext/blob/develop/erpnext/selling/doctype/customer/customer.json#L1377

Either ways, just remove the patch, will merge it. You still need to update framework.sql with "color" property.

Reply:
@rmehta Deleted patch and updated framework.sql